docs/reference/monitoring/troubleshooting.asciidoc

```diff
@@ -27,3 +31,26 @@ monitoring data by using {metricbeat} the indices have `-mb` in their names. If
 the indices do not exist, review your configuration. For example, see
 <<monitoring-production>>.
 
+[discrete]
+[[monitoring-troubleshooting-uuid]]
+=== Monitoring data for some {stack} nodes or instances is missing from {kib}
+
+*Symptoms*:
+The *Stack Monitoring* page in {kib} does not show information for some nodes or 
+instances in your cluster.
+
+*Resolution*:
+Verify that the missing items have unique UUIDs. Each {es} node, {ls} node,
+{kib} instance, and Beat instance is considered unique based on its persistent 
+UUID, which is found in its `path.data` directory. Alternatively, you can find
+the UUIDs in the product logs at startup.
+
+For {ls} nodes, you can also determine the UUID from the `id` property returned
+by the {logstash-ref}/monitoring-logstash.html[monitoring APIs root resource].
+
+
+TIP: When you install {kib} or Beat instances, do not copy existing content into
```

Comment:
A couple comments:

1. I think this applies to all 5 monitorable stack products today: Kibana, any of the Beats, Logstash, Elasticsearch, and APM server. When they are installed, their `path.data` directories should either be empty or non-existent; they should not be copied over from other installs.